start.sh

```diff
@@ -25,7 +25,7 @@ else
     # wget -O "${APP_DIR}/app_env" "${CONFIG_URL}/${ENVIRONMENT}/${APP_NAME}/env"
```

Comment:
I believe we need to uncomment this too, and delete the NOTE: about leaving it commented until 29 is done :)

Reply:
Doh! missed that. These new glasses aren't working